src/Monad/Listt/functions.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Widmogrod\Monad\Listt;
+
+use Widmogrod\Functional as f;
+use Widmogrod\Monad\Maybe as m;
+use Widmogrod\Primitive\Listt;
+
+/**
+ * @var callable
+ */
+const mempty = 'Widmogrod\Monad\Listt\mempty';
+
+/**
+ * Get an empty Listt.
+ *
+ * @return Listt
+ */
+function mempty(): Listt
+{
+    return Listt::mempty();
+}
+
+/**
+ * @var callable
+ */
+const push = 'Widmogrod\Monad\Listt\push';
+
+/**
+ * push :: [a] -> [a] -> [a]
+ *
+ * Append Listt with values.
+ *
+ * @param Listt    $list
+ * @param iterable $values
+ *
+ * @return Listt
+ *
+ * @throws \Widmogrod\Primitive\TypeMismatchError
+ */
+function push(Listt $list, iterable $values): Listt
+{
+    return $list->concat(fromRecursive($values));
+}
+
+/**
+ * @var callable
+ */
+const append = 'Widmogrod\Monad\Listt\append';
+
+/**
+ * append :: [a] -> a -> [a]
+ *
+ * @param Listt $list
+ * @param mixed $value
+ *
+ * @return Listt
+ */
+function append(Listt $list, $value = null): Listt
+{
+    return \call_user_func_array(f\curryN(2, function ($list, $value) {
+        return push($list, $value);
+    }), \func_get_args());
+}
+
+/**
+ * @var callable
+ */
+const head = 'Widmogrod\Monad\Listt\head';
+
+/**
+ * head :: [a] -> a
+ *
+ * Return head of a Listt
+ *
+ * @param \Widmogrod\Primitive\Listt $list
+ *
+ * @return Listt
+ */
+function head(Listt $list): Listt
+{
+    return $list->head();
+}
+
+/**
+ * @var callable
+ */
+const headMaybe = 'Widmogrod\Monad\Listt\headMaybe';
+
+/**
+ * headMaybe :: [a] -> Maybe a
+ *
+ * Return Maybe head of a Listt
+ *
+ * @param \Widmogrod\Primitive\Listt $list
+ *
+ * @return m\Maybe
+ */
+function headMaybe(Listt $list): m\Maybe
```

Comment:
Please import whole namespace for Maybe.
This will make code more readable.
For functions from maybe package `m\somthing` alias can stay.